.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+      - staging
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+
+    - name: Run Python deps
+      run: |
+        pip install pipenv
```

Comment:
could we add this piece in makefile? `make python-deps` (something similar)

Reply:
sure thing

Reply:
done